### PR TITLE
Adding testing and installation instructions to README

### PR DESCRIPTION
**Pull request by pabloem**

### CODE REVIEW

**Comment by pabloem** [comment]
r: @iasoon

File: README.md [hunk 1]
```diff
@@ -1,3 +1,50 @@
...
+
+To install the existing Ray Beam runner from a clone of this repository, you can follow the next steps:
+
+```shell
+# First create a virtual environment to install and run Python dependencies
+virtualenv venv
+. venv/bin/activate
+
+# Install development dependencies for the project
+pip install -r requirements_dev.txt
```
Comment (iasoon) [inline README.md:21]: Is this step still needed, or should everything be in the test dependencies already?

**Review by iasoon — APPROVED** [review]
Nice! We needed this. Thanks Pablo!

File: README.md (hunk 1 above)
Comment (pabloem) [reply README.md:21]: we still need this to install the autoformaters and checkers (E.g. black, flake8)

**Comment by pabloem** [comment]
thanks @iasoon ! merging as we do still need the requirements file installation.

File: README.md (hunk 1 above)
Comment (iasoon) [reply README.md:21]: Ah, yes, of course :)

**Pull request merged**